script/Arcueid/Day 9/09_00_ARC09_1.txt

```diff
@@ -296,7 +305,9 @@ Everything was so ridiculous that it's hard for me to put my thoughts in order.
 [sha:f7cfb148f149e2f3378b545e08cc0c5bbe400cc3]{
 -- Page 14, Offset 42659. Choice.
 -- 黒い男の事だ。
- the man in black.
+// The text referencing this guy in the moment was much more shadow oriented
+// I also refuse to allow this to become a ~(white woman) spotted
+ the shadowy figure.
```

Comment:
is this talking about Gouto? How do we describe him elsewhere?

Reply:
It's talking about whoever attacked Shiki at the end of D8 and got iced by Noel. If that's Gouto (it did mention one visible eye :thinking:) then this is Gouto.